test/command_line/helper/command_runner.rb

```diff
@@ -105,17 +105,20 @@ def grndb(command, *arguments)
   def find_program(name, options={})
     ENV["PATH"].split(File::PATH_SEPARATOR).each do |path|
       program_path = File.join(path, name)
+      program_exe_path = [path, name].join(File::ALT_SEPARATOR || File::SEPARATOR) + ".exe"
```

Comment:
We can use `RbConfig::CONFIG"EXEEXT"]` instead of `".exe"` literal.

Reply:
Should we need to use `File::ALT_SEPARATOR`?
Does this work?

```ruby
program_path = File.join(path, name) + RbConfig::CONFIG["EXEEXT"]
```

Reply:
No, some grndb test case fails because  of `File::SEPARATOR`.
Typically, full path of ".../grndb.exe"  and "...\\grndb.exe" are diffrent.

```
Failure: test_corrupt_double_array_table(TestGrnDBCheck)
c:/work/groonga/test/command_line/suite/grndb/test_check.rb:208:in `test_corrupt_double_array_table'
     205:     error = assert_raise(CommandRunner::Error) do
     206:       grndb("check")
     207:     end
  => 208:     assert_equal(<<-MESSAGE, error.error_output)
     209: [Users] Table is corrupt. (1) Truncate the table (truncate Users or '#{real_grndb_path} recover --force-truncate #{@database_path}') and (2) load data again.
     210:     MESSAGE
     211:   end
<"[Users] Table is corrupt. (1) Truncate the table (truncate Users or 'c:\\\\work\\インストール\\groonga-9.0.0-x64-2019-02-15\\bin/grndb.exe recover --force-truncate c:/work/groonga/test/command_line/tmp/groonga-command-line/test_corrupt_double_array_table-TestGrnDBCheck-.db') and (2) load data again.\n"> expected but was
<"[Users] Table is corrupt. (1) Truncate the table (truncate Users or 'c:\\\\work\\インストール\\groonga-9.0.0-x64-2019-02-15\\bin\\grndb.exe recover --force-truncate c:/work/groonga/test/command_line/tmp/groonga-command-line/test_corrupt_double_array_table-TestGrnDBCheck-.db') and (2) load data again.\n">
```

Reply:
Ah, I see. Thanks.